integrations/google-mail/nango.yaml

```diff
@@ -8,28 +8,42 @@ integrations:
                     but metadata can be set using the `backfillPeriodMs` property
                     to change the lookback. The property should be set in milliseconds.
                 input: OptionalBackfillSetting
-                version: 1.0.1
+                version: 1.0.2
+                scopes:
+                    - https://www.googleapis.com/auth/gmail.readonly
                 output: GmailEmail
                 sync_type: incremental
-                endpoint: GET /google-mail/emails
+                endpoint: GET /emails
         actions:
             send-email:
-                scopes: send
                 input: GmailEmailInput
                 output: GmailEmailSentOutput
-                endpoint: POST /google-mail/send
+                scopes:
+                    - https://www.googleapis.com/auth/gmail.send
+                endpoint: POST /send
                 description: |
                     Send an Email using Gmail.
+            fetch-document:
+                input: Metadata
```

Comment:
`Metadata` is not a good name considering we use that name for syncs. Could just be DocumentInput.

Reply:
Updated with the latest commit 🙏